articles/watch-history1.md

```diff
@@ -0,0 +1,55 @@
+---
+title:  Transferring your Instagram post/video viewing history elsewhere 
+datatype: "Viewing History"
+sources: Instagram
+destinations: ["Download myself", "Google Drive", "Dropbox"], 
```

Comment:
Heads up @lisad, this trailing comma is apparently not valid syntax so the portmap deploy failed: https://github.com/dtinit/portmap/actions/runs/9587501977/job/26437681020#step:8:171